azure-pipelines.yml

```diff
@@ -39,6 +39,8 @@ variables:
     value: "Tiberius"
```

Comment:
I've put this comment here because GitHub won't let me add comments to unchanged code :)

**Deployment\templates\continuous-testing.yml**
UseDotNet@2 task. Change to match the ones in azure-pipelines.yml so it uses the variable for the dotnet version.

**NVDSuppressions.xml**
Remove unused suppressions.

**Deployment\Modules\Webapp\main.tf**
This still contains references to .NET 6.


**Assorted warnings:**

**UKHO.ERPFacade.API\Program.cs**
app.UseElasticApm(configuration);
This is deprecated. See how it's done in FSS API for replacement setup.

**UKHO.ERPFacade.API\UKHO.ERPFacade.API.csproj**
Remove duplicate reference for Microsoft.Extensions.Diagnostics.HealthChecks.

**UKHO.ERPFacade.API.FunctionalTests\Helpers\ADAuthTokenProvider.cs**
WithAuthority is deprecated in line 43. Replace code (see UKHO.FileShareService.API\UKHO.FileShareService.API.Functional.AutoTests\Helper\AuthTokenProvider.cs in FSS API for an example).

**UKHO.ERPFacade.Common\HttpClients\EESClient.cs**
Remove unused variable _eesHealthCheckEnvironmentConfiguration.

Reply:
**Deployment\templates\continuous-testing.yml**
This is ok, but I don't think you needed to pass the .NET version through as a parameter. You should still be able to refer to the variable declared in azure-pipelines.yml.

**Deployment\Modules\Webapp\main.tf**
This still contains a reference to .NET 6 (line 51).

**UKHO.ERPFacade.API.FunctionalTests\Helpers\ADAuthTokenProvider.cs**
I'm slightly suprised this is working :) but I think it should also have the tenant added into line 39:
WithRedirectUri("http://localhost")**.WithTenantId(Config.TestConfig.AzureADConfiguration.TenantId)**.Build();